src/dev/i18n/extractors/html.js

```diff
@@ -112,7 +112,7 @@ function parseIdExpression(expression) {
     }
   }
 
-  return null;
+  throw createFailError(`Message id should be a string literal: \n${expression}`);
```

Comment:
__nit:__ `Message id should be a string literal:` ---> `Message id should be a string literal. but got:`